script/verification/Verification.s.sol

```diff
@@ -21,7 +22,7 @@ contract VerificationBase {
     }
 }
 
-contract SuperchainRegistry is CommonBase {
+contract SuperchainRegistry is ScriptBase {
```

Comment:
what's the difference? why don't we want or need this for all other script contacts?

Reply:
I think I may be able to revert this. I had it previously because it exposed `external` TOML utilities which I could wrap with a try/catch.

Reply:
I reverted this change in the end.